dist/index.html

```diff
@@ -57,7 +58,7 @@ <h1>Please Log In</h1>
       <table class="table">
         <thead>
           <tr>
```

Comment:
I like this! Wish I would have done a table here instead of displaying grid, had lots of design issues!